.circleci/config.yml

```diff
@@ -492,6 +492,7 @@ workflows:
           requires:
             - ktlint_check
             - detekt
+      - firebase_screenshots
```

Comment:
should be removed

Reply:
It's removed now. It just was present because I had to regenerate the screenshots. For that cause I also added the "do not merge" label again. But now it can be merged